services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/ReadBatch.java

```diff
@@ -37,14 +45,26 @@ private ReadBatch(BuilderImpl<?> builder) {
         this.keysAndAttributes = generateKeysAndAttributes(builder.requests, builder.mappedTableResource);
     }
 
+    /**
+     * All objects must be constructed using a Builder.
+     * @param itemClass to determine the parameterized type T of the builder
+     * @param <T> The type of the modelled object, corresponding to itemClass
+     * @return a parameterized builder of this type
```

Comment:
Anywhere you write 'parameterized' we should probably rewrite that. It's not a usability friendly term in my opinion.

Reply:
Changed